octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/mywork/MyWorkView.java

```diff
@@ -107,7 +117,18 @@ protected IStatus run(IProgressMonitor monitor) {
                         errorComposite.setErrorMessage("Error while loading \"My Work\": " + e.getMessage());
                         showControl(errorComposite);
                         entityData.setEntityList(Collections.emptyList());
-                    });
+                        
+                        Display.getDefault().asyncExec(() -> {
+                            Activator.setActiveItem(null);
+                            new InfoPopup(
+                                    "My work failed to load",

```

Comment:
We should use similar connection settings to what we have in IntelliJ:
                                    "Your previously saved connection settings do not seem to work",
                                    "Please go to settings and test your connection to Octane",
                                    "Retry to connect Octane");